examples/deployment-with-init/deployment.tf

```
@@ -0,0 +1,51 @@
+
+# Create an Elastic Cloud deployment
+resource "ec_deployment" "example_minimal" {
+  # Optional name.
+  name = "my_example_deployment"
+
+  # Mandatory fields
+  region                 = "us-east-1"
+  version                = "7.9.2"
+  deployment_template_id = "aws-io-optimized-v2"
+  traffic_filter = [ec_deployment_traffic_filter.allow_all.id]
+  elasticsearch {
+    topology {
+      instance_configuration_id = "aws.data.highio.i3"
+      memory_per_node="8g"
```

`memory_per_node` has been removed in favour of `size`.
```suggestion
      size = "8g"
```